{
  "uid" : "766ffb93b425a6c4",
  "name" : "adds correctly if it does not overflow and the result is positive",
  "fullName" : "Contract: SignedSafeMath add.adds correctly if it does not overflow and the result is positive",
  "historyId" : "db4ec3f137ce67cb3d8e8eb3319a6208",
  "time" : {
    "start" : 1635554934676,
    "stop" : 1635554935266,
    "duration" : 590
  },
  "description" : "",
  "descriptionHtml" : "",
  "status" : "broken",
  "statusMessage" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0684160>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
  "statusTrace" : "Error: Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0684160>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))\n                    at Context.<anonymous> (test/utils/math/SignedSafeMath.test.js:10:49)\n                    at processImmediate (node:internal/timers:464:21)",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "suite",
    "value" : "Contract: SignedSafeMath add"
  }, {
    "name" : "testClass",
    "value" : "Contract: SignedSafeMath add"
  }, {
    "name" : "testMethod",
    "value" : "adds correctly if it does not overflow and the result is positive"
  }, {
    "name" : "package",
    "value" : "Contract: SignedSafeMath add"
  }, {
    "name" : "resultFormat",
    "value" : "allure1"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : true,
  "retry" : true,
  "extra" : {
    "categories" : [ ],
    "tags" : [ ]
  },
  "source" : "766ffb93b425a6c4.json",
  "parameterValues" : [ ]
}